tools/vargo/src/main.rs

```diff
@@ -679,6 +683,111 @@ fn run() -> Result<(), String> {
+                            return Ok(());
+                        }
+
+                        let verusfmt_check = match std::process::Command::new("verusfmt")
+                            .arg("--version")
+                            .output()
+                        {
+                            Ok(output) => {
+                                if !output.status.success() {
+                                    return Err(format!(
+                                        "`verusfmt` returned status code {:?}",
+                                        status.code()
+                                    ));
+                                }
+                                let verusfmt_version_stdout = String::from_utf8(output.stdout)
+                                    .map_err(|_| format!("invalid output from verusfmt"))?;
+                                let verusfmt_version_re =
+                                    Regex::new(r"^verusfmt ([0-9]+)\.([0-9]+)\.([0-9]+)\n$")
+                                        .unwrap();
+                                let verusfmt_version = verusfmt_version_re
+                                    .captures(&verusfmt_version_stdout)
+                                    .ok_or(format!("invalid output from verusfmt"))?
+                                    .extract::<3>()
+                                    .1
+                                    .iter()
+                                    .map(|v| v.parse::<u64>().unwrap())
+                                    .collect::<Vec<u64>>();
+                                for (cv, ev) in
+                                    verusfmt_version.iter().zip(MINIMUM_VERUSFMT_VERSION.iter())
+                                {
+                                    if ev > cv {
+                                        return Err(format!("expected `verusfmt` version to be at least {}.{}.{}, found {}.{}.{}; refer to CONTRIBUTING.md for installation instructions",
+                                            MINIMUM_VERUSFMT_VERSION[0], MINIMUM_VERUSFMT_VERSION[1], MINIMUM_VERUSFMT_VERSION[2],
+                                            verusfmt_version[0], verusfmt_version[1], verusfmt_version[2]));
+                                    } else if cv > ev {
+                                        break;
+                                    }
+                                }
+                                let original_args = original_args
+                                    .iter()
+                                    .map(|x| x.as_str())
+                                    .collect::<std::collections::HashSet<&str>>();
+                                if !original_args.is_subset(&std::collections::HashSet::from([
+                                    "--",
+                                    "--check",
+                                    "fmt",
+                                    "--color=always",
```

Comment:
I'm not sure why this is here, since:
```console
$ vargo fmt -- --color=always
Option 'color' given more than once
Option 'color' given more than once
error: `cargo fmt` returned status code Some(1)

$ vargo fmt --color=always
error: unexpected argument '--color' found

  tip: to pass '--color' as a value, use '-- --color'

Usage: cargo fmt [OPTIONS] [-- <rustfmt_options>...]

For more information, try '--help'.
error: `cargo fmt` returned status code Some(2)
```

Reply:
Because `--color=always` gets added earlier by `vargo` to `args`, so this would fail otherwise.

Reply:
Ah, thanks for the explanation!